kong.yaml

```
@@ -0,0 +1,84 @@
+package:
+  name: kong
+  version: 3.9.0
+  epoch: 0
+  description: "The Kong Gateway - an API Gateway built on Nginx and OpenResty"
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - perl
+
+environment:
+  contents:
+    packages:
+      - bash
+      - bazel
+      - bubblewrap
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - curl
+      - expat-dev
+      - jq
+      - libelf
+      - lua5.1-mpack
+      - luajit-dev
+      - openjdk-21
+      - openssl
+      - openssl-dev
+      - patch
+      - pcre2-dev
+      - perl
+      - pkgconf
+      - procps
+      - protobuf-dev
+      - rust
+      - valgrind-dev
+      - yaml-dev
+      - zlib-dev
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/Kong/kong
+      tag: ${{package.version}}
+      expected-commit: 0118b3c220681d4f0327540ae1a5e8f271081bf6
+
```

And before applying the patch, replace the dummy (`KONG_MANAGER_VERSION`) (you may need GNU grep):

```suggestion

    - runs: |
        KONG_MANAGER_VERSION=$(grep -oP '^KONG_MANAGER=\K.*' .requirements)
        sed -i "s/KONG_MANAGER_VERSION/${KONG_MANAGER_VERSION}/g" http-archive.patch

```